qemu/tests/rng_bat.py

```diff
@@ -94,8 +103,7 @@ def is_dev_used_by_qemu(dev_file, vm_pid):
     if os_type == "linux":
         check_rngd_service = params.get("check_rngd_service")
         if check_rngd_service:
-            output = session.cmd_output(check_rngd_service)
-            if 'running' not in output:
+            if not utils_misc.wait_for(_check_rngd_status, 30):
```

Comment:
Suggest using `_is_rngd_running` instead of `_check_rngd_status`.
And from the purpose for this fix, wait a short time before start checking is better,  means increase `first` for `utils_misc.wait_for`.

Reply:
Updated, thanks!